.github/workflows/push-main.yml

```diff
@@ -103,18 +74,29 @@ jobs:
           workload_identity_provider: ${{env.WORKLOAD_IDENTITY_POOL_ID}}
           service_account: ${{env.IAM_SERVICE_ACCOUNT}}
 
-      - name: ✍🏽 Login to GAR
+      - name: ✍🏽 Login to GAR using skopeo
         run: |
           sudo skopeo login -u oauth2accesstoken --password=${{ steps.auth.outputs.access_token }} https://${{env.GAR_IMAGE_REGISTRY}}
 
-      - name: 🐳 Sync combined images to Artifact Registry
+      - name: ✍🏽 Login to dockerhub using skopeo
+        env:
+          docker_user: ${{ secrets.DOCKERHUB_USER_NAME }}
+          docker_password: ${{ secrets.DOCKERHUB_ACCESS_TOKEN }}
         run: |
-          sudo skopeo sync \
-            --src-tls-verify=false \
-            --registries-conf=.github/registries.conf \
-            --src yaml \
-            --dest docker \
-            .github/sync-containers.yml ${{env.GAR_IMAGE_REGISTRY}}/gitpod-artifacts/docker-dev
+          sudo skopeo login -u ${{ env.docker_user }} --password=${{ env.docker_password }} https://${{ env.DH_IMAGE_REGISTRY }}
```

Comment:
Why was the login for Docker Hub moved up to here? Ideally, this would not be done, like if a prior step fails.

Reply:
[force-pushed](https://github.com/gitpod-io/workspace-images/compare/0fa3a34529477a37edfd9657599a52442dc04f0b..0d45b7c3f50a76fc3bd304298906131c77773c86) the prs/gar-cache branch from [0fa3a34 ](https://github.com/gitpod-io/workspace-images/commit/0fa3a34529477a37edfd9657599a52442dc04f0b)to [0d45b7c ](https://github.com/gitpod-io/workspace-images/commit/0d45b7c3f50a76fc3bd304298906131c77773c86)[now](https://github.com/gitpod-io/workspace-images/pull/718#event-6121426267) to fix